UPDATING.md

```diff
@@ -23,6 +23,8 @@ assists people when migrating to a new version.
 
 ## Next
 
+* [10324](https://github.com/apache/incubator-superset/pull/10324): Facebook Prophet has been introduced as an optional dependency to add support for timeseries forecasting in the chart data API. To enable this feature, install Superset with the optional dependency `prophet` or directly `pip install fbprophet`.
```

Comment:
how to install Superset with the optional dependency `prophet`? i have already installed superset. Now how to install prophet to my existing superset?

Reply:
Hi @wakilkhan96

If you haven't managed to solve installing prophet, access the terminal for your **superset_app** and run the following commands.

`pip uninstall fbprophet pystan`
`pip --no-cache-dir install pystan==2.19.1.1`
`pip install prophet`

That should solve that issue.
 # pystan>=3.0 is currently not supported for prophet that why you should specify that version.